Loop/Managers/Alerts/UserNotificationAlertIssuer.swift

```diff
@@ -68,7 +68,10 @@ fileprivate extension Alert {
         let userNotificationContent = UNMutableNotificationContent()
         userNotificationContent.title = content.title
         userNotificationContent.body = content.body
-        userNotificationContent.sound = getUserNotificationSound()
+        userNotificationContent.sound = userNotificationSound
+        if #available(iOS 15.0, *) {
+            userNotificationContent.interruptionLevel = backgroundContent?.isCritical == true ? .critical : .timeSensitive
```

Comment:
[nit] Or are all notifications time sensitive? I'm not sure every notification should be time sensitive. Perhaps I'm just misunderstanding this.

Reply:
@darinkrauss yes, for now.  There are more LOOP tickets on the way to address being more selective.